funnel/views/update.py

```diff
@@ -31,24 +32,28 @@
 @route('/<profile>/<project>/updates')
 class ProjectUpdatesView(ProjectViewMixin, UrlChangeCheck, UrlForView, ModelView):
     @route('', methods=['GET'])
-    @render_with('project_updates.html.jinja2', json=True)
+    @render_with(html_in_json('project_updates.html.jinja2'))
     @requires_roles({'reader'})
     def updates(self):
-        return {
-            'project': self.obj.current_access(datasets=('primary', 'related')),
-            'draft_updates': (
-                [
-                    update.current_access(datasets=('primary', 'related'))
-                    for update in self.obj.draft_updates
-                ]
-                if self.obj.features.post_update()
-                else []
-            ),
-            'published_updates': [
+        project = self.obj.current_access(datasets=('primary', 'related'))
+        draft_updates = (
+            [
                 update.current_access(datasets=('primary', 'related'))
```

Comment:
Here instead of `primary` dataset it should be `without_parent`, as primary will include the project's data with each update.